Homework - Functions.py

```diff
@@ -0,0 +1,55 @@
+# Exercises 1:
+
+def my_function(a=1, b=5, c=-3, d='abc', e=[12, 56, 'cad']):
+    return a + b - c
+
+print(my_function(1, 5, 3))
+
+def my_function():
+    return 0
+
+print(my_function())
+
+def my_function(a=2, b=4, c='abc', param_1=2):
```

Comment:
This one should be only one function. You're not suppose to create one function for each example, but one function that works for all those examples (and others).